grails-app/conf/Config.groovy

```diff
@@ -221,7 +221,8 @@ portal {
     siteHeader = "Open Access to Ocean Data"
     motdUrl = "https://static.emii.org.au/motd"
 
-    logo = "images/IMOS_logo.png"
+    logo = "images/AODN/AODN_logo.png"
+    secondaryLogo ="https://static.emii.org.au/images/logo/IMOS_logo-stacked3.png"
```

Comment:
Would be good if we could remove IMOS specific stuff from the base code.  Not including an IMOS specific logo here would be a good start.